SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -473,6 +444,24 @@ bool FLocalDeploymentManager::StartLocalDeploymentShutDown()
 	return true;
 }
 
+bool FLocalDeploymentManager::WaitForGracefulShutdown()
+{
+#if PLATFORM_WINDOWS
+	WindowsRequestGracefulRuntimeShutdown();
```

Comment:
Nit: Would prefer if these were explicit on how they shut the runtime down. For example ShutdownRuntimeWithProcessSignal / ShutdownRuntimeWithHTTPRequest. What do you think?

Reply:
Changed to use `TryGracefullyKill` and `TryGracefullyKillWindows` within spatialcommandutils in updated PR, I think the windows name makes more sense with this update as now both methods send a signal?